presto-server-rpm/pom.xml

```diff
@@ -83,7 +83,7 @@
             <plugin>
                 <groupId>ca.vanzyl.provisio.maven.plugins</groupId>
                 <artifactId>provisio-maven-plugin</artifactId>
-                <version>1.0.8</version>
+                <version>${dep.provisio.version}</version>
```

Comment:
No need to specify this version in module if it is already defined in root pom.

Reply:
Moved to pluginManagement